dm/openapi/fixtures/task.go

```diff
@@ -53,8 +53,45 @@ var (
 		  "user": "root"
 		},
 		"task_mode": "all"
-	  }
-`
+	}
+	`
+
+	noShardErrNameJSONStr = `
+	{
+		"enhance_online_schema_change": true,
+		"meta_schema": "dm_meta",
+		"name": "orders_netpay_pay_inf_nuccinterconn_zh_ext_2022_[6-12]",
```

Comment:
Do we need to change the task name into a more generic name, and not hard-coded ?  For example, put this variable inside `GenNoShardErrNameOpenAPITaskForTest` as a string template, and generate the task name in a random sequence of letters with specific length, and generate the final task.

Reply:
I think it is difficult to put the name in this multi-lined string. But I think we can use the default NoShard task and manually assign it into task.Name